controller:
  type: "joint_state_controller/JointStateController"
  publish_rate: 50

stationary_base_spin_base_joint_controller:
  type: "position_controllers/JointPositionController"
  joint: stationary_base_spin_base_joint
spin_base_arm1_joint_controller:
  type: "position_controllers/JointPositionController"
  joint: spin_base_arm1_joint
arm1_arm2_joint_controller:
  type: "position_controllers/JointPositionController"
  joint: arm1_arm2_joint
arm2_arm3_joint_controller:
  type: "position_controllers/JointPositionController"
  joint: arm2_arm3_joint